spring-boot-project/spring-boot-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -251,17 +251,17 @@ endif::[]
 
 [[getting-started-installing-the-cli]]
 === Installing the Spring Boot CLI
-The Spring Boot CLI is a command line tool that can be used if you want to quickly
-prototype with Spring. It allows you to run http://groovy-lang.org/[Groovy] scripts,
-which means that you have a familiar Java-like syntax, without so much boilerplate code.
+The Spring Boot CLI (Command Line Interface) is a command line tool that you can use to quickly
```

Comment:
What's the policy for expanding CLI to Command Line Interface? It was already done [above](https://github.com/spring-projects/spring-boot/pull/10763/files#diff-5c18fc93d7063a97159926b2a132dc13R77) but then isn't done below. I'm just curious so that we can follow the convention in the future.

Reply:
The policy is to define it the first time it's used and then periodically throughout the document. I did define it at the first instance in the document (currently line 77). Given that the term appears in the heading here, I thought defining it here too would be a good idea.

Reply:
Thanks